unittest/cdomains/floatDomainTest.ml

```diff
@@ -189,7 +189,7 @@ struct
     begin
       check_meet (FI.top ()) (FI.top ()) (FI.top ());
       check_meet (FI.top ()) fi_one fi_one;
-      assert_raises (Failure "meet results in empty Interval") (fun () -> (FI.meet fi_zero fi_one));
+      check_meet fi_zero fi_one (FI.bot ());

```

Comment:
We might want to check for the occurrence of the exception in the arithmetic operations then

Reply:
Well the meet does not cause an exception, so it does not occur. The exception only occurs when we do arithmetic on `Bot` (add, mul, any lib-function, etc). But we can also add tests for arithmetic on `Bot`, it has nothing to do with `meet` however

Reply:
That's what i meant

Reply:
ah, ok, I will do